packages/device_info_plus/device_info_plus/android/src/main/kotlin/dev/fluttercommunity/plus/device_info/MethodCallHandlerImpl.kt

```diff
@@ -37,6 +40,7 @@ internal class MethodCallHandlerImpl(
             build["manufacturer"] = Build.MANUFACTURER
             build["model"] = Build.MODEL
             build["product"] = Build.PRODUCT
+            build["name"] = Settings.Global.getString(context.contentResolver, Settings.Global.DEVICE_NAME)
```

Comment:
This API is only available from Android N (API 25), this code needs to check for the version. Android Studio should show a warning.